nix/flake.nix

```diff
@@ -144,7 +144,8 @@
               cp -r assets/macos/WezTerm.app "$OUT_APP"
               rm $OUT_APP/*.dylib
               cp -r assets/shell-integration/* "$OUT_APP"
-              ln -s $out/bin/{wezterm,wezterm-mux-server,wezterm-gui,strip-ansi-escapes} "$OUT_APP"
+              mv $out/bin/{wezterm,wezterm-mux-server,wezterm-gui,strip-ansi-escapes} "$OUT_APP"
```

Comment:
Please add a comment explaining why we're doing it this way 🙏

Reply:
You got it.

Reply:
Should something equivalent happen over in the `ci/deploy.sh` where the main part of the build assembles its bundle?

https://github.com/wezterm/wezterm/blob/f06d6fc04fd55995dd4c0e7c8cf1bee5c63bc005/ci/deploy.sh#L22-L53

Reply:
It appears that already creates the binaries directly in the bundle structure (via cp or lipo) rather than creating symbolic links, so I do not think any adjustment is needed. And indeed, the panic does not happen when using the nightly.